File: Api/Entities/Address.cs
namespace Api.Entities
{
    public class Address
    {
        public int Id { get; set; }
        public string UserAddress { get; set; }
        public string Country { get; set; }
        public string City { get; set; }
        public bool IsMain { get; set; }
        public string Phone { get; set; }

        public int UserId { get; set; }

        public User User { get; set; }

    }
}

File: client/src/app/model/Address.ts
export interface ISetAddress {
    UserAddress: string;
    Country: string;
    City: string;
    Phone: string;
}


export interface IAddress extends ISetAddress {
    id: number;
    userAddress: string;
    country: string;
    city: string;
    phone: string;
    isMain: boolean;
}
